jbpm-benchmarks/jbpm-performance-tests/src/main/java/org/jbpm/test/performance/jbpm/JBPMController.java

```diff
@@ -303,13 +271,14 @@ public static PoolingDataSource setupPoolingDataSource(Properties dsProps, Strin
                 for (String propertyName : new String[] { "serverName", "portNumber", "databaseName" }) {
                     pds.getDriverProperties().put(propertyName, dsProps.getProperty(propertyName));
                 }
-                pds.getDriverProperties().put("URL", dsProps.getProperty("url"));
+                pds.getDriverProperties().put("URL", dsProps.getProperty("connectionUrl"));
                 pds.getDriverProperties().put("selectMethod", "cursor");
                 pds.getDriverProperties().put("InstanceName", "MSSQL01");
             } else if (driverClass.startsWith("com.mysql")) {
```

Comment:
Well, I don't want to change anything here since in a foreseeable future we will migrate to Narayana anyway so I think any optimization would be useless here, I will still need to rewrite it. Does that make sense?

Reply:
It does, I completely agree.